CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 
 [All changes in [[UnreleasedVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.23.0...HEAD).
 
+### What's changed
```

Comment:
Should this actually be listed as a breaking change?  Code that worked before will no longer work.  That said, [we specifically say to call the macro in `lib.rs`](https://mozilla.github.io/uniffi-rs/tutorial/Rust_scaffolding.html#rust-scaffolding-code), so that code was only accidentally working.